design-system/molecules/cards/card-project-marketplace/adapters/default/default.adapter.tsx

```
@@ -1,9 +1,26 @@
 "use client";
 
-import { GitFork, Star, UserRound } from "lucide-react";
-import { ElementType, useMemo, useRef } from "react";
+import {
+  Coins,
+  Flame,
+  GitFork,
+  Hammer,
+  HandCoins,
+  Map,
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)